### PR TITLE
Added a page on azure vm extension

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

https://deploy-preview-1470--logz-docs.netlify.app/shipping/log-sources/azure-vm-extension.html
----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Comment by Doron-Bargo** [comment]
I dont like when we link to other companies pages
We should have the same documentation as Azure has on our extension

File: _source/logzio_collections/_log-sources/azure-vm-extension.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+shipping-tags:
+  -  azure
+order: 460
+---
+
+Extensions are small applications that provide post-deployment configuration and automation on Azure VMs. You can install Logz.io agents on Azure virtual machines as an extension. This will allow you to ship logs directly from your VM to your Logz.io account.
+
+
+<div class="tasklist">
+
```
Comment (Doron-Bargo) [inline _source/logzio_collections/_log-sources/azure-vm-extension.md:20]: need to add a note that we support only linux

File: _source/logzio_collections/_log-sources/azure-vm-extension.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+Extensions are small applications that provide post-deployment configuration and automation on Azure VMs. You can install Logz.io agents on Azure virtual machines as an extension. This will allow you to ship logs directly from your VM to your Logz.io account.
+
+
+<div class="tasklist">
+  
+##### Login to Logz.io from your Azure account
+  
+Log in to Logz.io using either the SSO in your Azure account or a link to the Logz.io platform.
```
Comment (Doron-Bargo) [inline _source/logzio_collections/_log-sources/azure-vm-extension.md:23]: Can you add pictures?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
The flow looks ok
Please see my comments
Also - @mirii1994 can you give your opinion ?

**Review by mirii1994 — APPROVED** [review]
lgtm, like doron said- we need to add a note that we support linux, and some screen shots for the steps

**Pull request merged**